src/test/components/MarkerChart.test.js

```diff
@@ -361,6 +364,12 @@ describe('MarkerChart', function() {
         fireFullClick(getByText(stringOrRegexp));
       }
 
+      //There are two menu items of the same name
+      function clickOnSubMenuItem(stringOrRegexp, i) {
```

Comment:
I don't think getting an `i` value and then clicking to it is a good way of testing. We should still do something like `stringOrRegexp` with `getByText`. Because index of this context menu is an implementation detail and that can change in the future if we decide to reorder or add another element to it.

Reply:
True. When using getByText, it can't figure out which 'From the start of this marker' we are looking for as they are two of them. 
Is there any other way to get it?

Reply:
I had a quick look and I have 2 suggestions.

1. Look up the parent, then the submenu.
2. Use the CSS class `react-contextmenu__visible` of its parent.

Suggestion 1 could look like this:
```
import { render, fireEvent, getByText as globalGetByText } from '@testing-library/react';
...
function clickOnSubMenuItem(parentLookup, submenuLookup) {
  const parent = getByText(parentLookup);
  const submenu = globalGetByText(parent.parentNode, submenuLookup);
  fireFullClick(submenu);
}
```

suggestion 2 could look like this:
```js
function clickOnSubMenuItem(stringOrRegexp) {
  const submenu = getByText(stringOrRegexp, { selector: '.react-contextmenu__visible > .react-contextmenu-item');
  fireFullClick(submenu);
}
```

You may have to use `await findByText` instead of `getByText` for solution 2, because the display of the menu doesn't seem synchronous.

Reply:
Actually looking at the code closer, I think that the suggestion 2 won't work unless we simulate hovering on the parent menu. This would be nice to do indeed, but I think we could still move forward with an unperfect test so that we move this patch forward. What do you all think?

Reply:
I have been trying to figure this out as well, but parent menu can't be fetched and if we do it without the parent menu.They are two of them.
So, yea let's move forward.

Reply:
I implemented suggestion 1 :-)